src/lib.rs

```diff
@@ -1,4 +1,10 @@
 #![doc(html_root_url = "https://docs.rs/prost/0.6.1")]
+#![no_std]
+
+extern crate alloc;
```

Comment:
likewise, I'm surprised this is necessary.

Reply:
i think in this case it's really necessary